scripts/target.sh

```diff
@@ -61,6 +65,32 @@ get_available_targets() {
     printf '%s\n' "${targets[@]}" | sort
 }
 
+# Returns 0 if the target supports the given architecture, 1 otherwise
+target_supports_arch() {
```

Comment:
```suggestion
target_supports_os() {
```

arch is more about amd64 vs arm

Reply:
I think only a couple or so teams make a relevant difference between OS and platform. 
That is not a necessary detail for now, so I'll stick with OS.